rapportannuel.html

```diff
@@ -0,0 +1,16 @@
+---
+title: "Incubateur de Services Numériques - Rapport Annuel"
+excerpt: "Depuis juin 2013, l’État expérimente au sein de la Direction Interministérielle du Numérique et des Systèmes d’Information et de Communication (DINSIC) une nouvelle manière de construire des services publics numériques."
+layout: embed
+permalink: /rapportannuel
+---
+
+<object class='embedded-document' data='rapportannuel.pdf' type='application/pdf'>
+    <iframe class='embedded-document' src='rapportannuel.pdf'>
+        <section id='rapportannuel' class='ui container'>
+            Votre navigateur ne semple pas supporter la lecture des fichiers PDF,
+            <br />
+            pour acceder au rapport annuel, veuillez cliquer <a class='primary' href='/rapportannuel.pdf'><strong>ici</strong></a>
```

Comment:
Yeah feel free to implement something simpler if you find a solution. I was just exploring the "let's embed the pdf with fallbacks". 😄